ansible/roles/dhcpd/tasks/main.yml

```diff
@@ -1,53 +1,71 @@
 ---
-# Copyright (C) 2014-2018 Maciej Delmanowski <drybjed@gmail.com>
-# Copyright (C) 2014-2018 DebOps <https://debops.org/>
-# SPDX-License-Identifier: GPL-3.0-only
+# Copyright (C) 2020 CipherMail B.V. <https://www.ciphermail.com/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
 
-- import_role:
-    name: 'secret'
+- name: Install ISC DHCP packages
+  package:
+    name: '{{ (dhcpd__base_packages + dhcpd__packages)|flatten }}'
+    state: 'present'
+  register: dhcpd__register_packages
+  until: dhcpd__register_packages is succeeded
 
-- include: dist_vars.yml
-- include: install.yml
-
-- name: Get list of nameservers configured in /etc/resolv.conf
-  shell: set -o nounset -o pipefail -o errexit &&
-         grep -E '^nameserver\s' /etc/resolv.conf | awk '{print $2}' || true
+- name: Divert original configuration
+  command: dpkg-divert --quiet --local
+           --divert {{ item }}.dpkg-divert
+           --rename {{ item }}
```

Comment:
Perhaps you could use the `dpkg_divert` module here instead of the `command` module?